tardis/io/config_reader.py

```diff
@@ -217,6 +217,10 @@ def from_yaml(cls, fname, *args, **kwargs):
         try:
             yaml_dict = yaml_load_file(fname,
                                        loader=kwargs.pop('loader', YAMLLoader))
+            start = yaml_dict['spectrum']['start'] 
+ 			stop = yaml_dict['spectrum']['stop'] 
+ 			num = yaml_dict['spectrum']['num'] 
+ 			yaml_dict['spectrum']['array'] = quantity_linspace(start, stop, num+1)       
```

Comment:
- Identation seems wrong.
- The bug report you say you are fixing wants the `start`, `stop`, and `num` properties removed, you still use them here and add another one.